<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1463</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mod buddy</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="clean_modfolder_button">
         <property name="text">
          <string>CLEAN UP MOD-FOLDER</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="initialize_mod">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>Execute current configuration</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="mod_tab">
       <attribute name="title">
        <string>Mod configuration</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QGroupBox" name="addmod_group">
             <property name="title">
              <string>Add mods</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QPushButton" name="new_mod_archived_button">
                <property name="text">
                 <string>Add mod from archive</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="new_mod_button">
                <property name="text">
                 <string>Add mod from folder</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="profile_group">
             <property name="title">
              <string>Profiles</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0" colspan="5">
               <widget class="QComboBox" name="profile_combobox"/>
              </item>
              <item row="2" column="0" colspan="2">
               <widget class="QPushButton" name="load_profile_button">
                <property name="text">
                 <string>Load profile</string>
                </property>
               </widget>
              </item>
              <item row="2" column="4">
               <widget class="QPushButton" name="duplicate_profile_button">
                <property name="text">
                 <string>Save profile as...</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2" colspan="2">
               <widget class="QPushButton" name="save_profile_button">
                <property name="text">
                 <string>Save profile</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="game_group">
             <property name="title">
              <string>Game</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="2" column="1">
               <widget class="QPushButton" name="load_game_button">
                <property name="text">
                 <string>Load game target</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QPushButton" name="new_game_button">
                <property name="text">
                 <string>New game</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QComboBox" name="game_combobox"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="main_component">
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
           <item>
            <widget class="QGroupBox" name="mods_component">
             <property name="title">
              <string>Mod table</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QTableView" name="mod_list">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>1</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::StrongFocus</enum>
                </property>
                <property name="contextMenuPolicy">
                 <enum>Qt::NoContextMenu</enum>
                </property>
                <property name="horizontalScrollBarPolicy">
                 <enum>Qt::ScrollBarAsNeeded</enum>
                </property>
                <property name="showDropIndicator" stdset="0">
                 <bool>false</bool>
                </property>
                <property name="dragDropMode">
                 <enum>QAbstractItemView::NoDragDrop</enum>
                </property>
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::SingleSelection</enum>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <property name="showGrid">
                 <bool>true</bool>
                </property>
                <property name="sortingEnabled">
                 <bool>false</bool>
                </property>
                <property name="cornerButtonEnabled">
                 <bool>true</bool>
                </property>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderCascadingSectionResizes">
                 <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderHighlightSections">
                 <bool>true</bool>
                </attribute>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="mod_controls" native="true">
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="move_up">
                   <property name="text">
                    <string>UP</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="toggle_mod">
                   <property name="text">
                    <string>TOGGLE</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="edit_mod">
                   <property name="text">
                    <string>EDIT</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="move_down">
                   <property name="text">
                    <string>DOWN</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="dir_component">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Target game</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QWidget" name="dest_component" native="true">
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="1" column="0">
                  <widget class="QLabel" name="mod_dest">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QTreeView" name="file_view"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="sources_tab">
       <attribute name="title">
        <string>Source configuration</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="source_groupbox">
          <property name="title">
           <string>Sources control</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="source_add">
             <property name="text">
              <string>Add source from URL</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="source_check_updates">
             <property name="text">
              <string>Check for updates</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="source_download">
             <property name="text">
              <string>Download outdated sources</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="source_edit">
             <property name="text">
              <string>Edit source...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="source_tableview"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1463</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
